research-app/src/App.vue

```diff
@@ -1648,13 +1737,85 @@ export default class App extends WWTAwareComponent {
     return imagesets.filter(iset => iset.name.toLowerCase().includes(searchText) || iset.description.toLowerCase().includes(searchText));
   }
 
+  // RA, Dec in radians
+  distance(pt1: AngleCoordinates, pt2: AngleCoordinates): number {
```

Comment:
This function would be a great candidate for moving into the `astro` package, which aims to bundle together generic astronomy-ish routines that don't depend on WWT-specific types.